### ISSUE TITLE
Chrome net::ERR_ABORTED

### ISSUE THREAD

**Issue by minhhieutran2112** [g1879/DrissionPage#429]
Hi,

This problem happened in Windows, DrissionPage v4.0.5.6, and also the newest v4.1.

**What happened:** After initiating the driver and do a driver.get, I got this error: net::ERR_ABORTED.

Is there anyway to resolve this please?

![image](https://github.com/user-attachments/assets/029149ab-b53b-423a-a49c-0a70d51ac743)

**Comment by qing762** [comment]
Yep, can confirm the issue. Been having it for three days or so.
